src/main/java/ro/societateahermes/backendservice/entities/CDMember.java

```diff
@@ -0,0 +1,44 @@
+package ro.societateahermes.backendservice.entities;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Pattern;
+
+@Data
+@Entity
+@NoArgsConstructor
+public class CDMember {
+    @Id
+    @GeneratedValue
+    private Long CDMemberID;
+    @NotEmpty(message = "Image path cannot be null!")
+    private String imagePath;
+    @NotEmpty(message = "Facebook link cannot be null!")
+    private String facebookLink;
+    @NotEmpty(message = "Description is mandatory!")
+    private String description;
+    @NotEmpty(message = "Name is mandatory!")
+    @Pattern(regexp = "[a-zA-Z]+", message = "Name cannot contain numbers!")
+    private String name;
+    @NotEmpty(message = "Position is mandatory!")
+    @Pattern(regexp = "[a-zA-Z]+", message = "Position cannot contain numbers!")
+    private String position;
+
+    public CDMember(Long CDMemberID, @NotEmpty(message = "Image path cannot be null!") String imagePath,
+                    @NotEmpty(message = "Facebook link cannot be null!") String facebookLink,
+                    @NotEmpty(message = "Description is mandatory!") String description,
+                    @NotEmpty(message = "Name is mandatory!") String name,
+                    @NotEmpty(message = "Position is mandatory!") String position) {
+        this.CDMemberID = CDMemberID;
+        this.description = description;
+        this.imagePath = imagePath;
+        this.name = name;
+        this.position = position;
+        this.facebookLink = facebookLink;
+    }
```

Comment:
foarte bine👍

Reply:
super tare <3 

Mica chestie de adaugat @mircea216 avand acel @NoEmpty sau @NotNull pe field, nu mai trebuie adaugat si in constructor, intrucat cand o sa puna valoarea, o sa valideze.

Also, validarile ar trebui sa se faca pe obiectul care vine de la front, nu pe cel care e salvat in DB, intru cat asta se presupune ar fi un obiect valid la momentul salvarii, deci te rog sa pui validarile pe DTO

Reply:
@fujinuji am lasat doar pe partea de DTO